pygeoapi/linked_data.py

```diff
@@ -180,26 +182,59 @@ def geojson2geojsonld(config, data, dataset, identifier=None):
 
     context = config['resources'][dataset].get('context', [])
 
-    uri = data['properties'].get('uri') if 'properties' in data else None
+    # Currently "uri" is a magic property, eventually we should have this
+    # be in the config so a user can override the use of a URI.
+    uri = data.get('properties', {}).get('uri', None)
 
     if identifier and uri:
         data['id'] = '{}'.format(uri)
     elif identifier:
-        data['id'] = '{}/collections/{}/items/{}'.format(config['server']['url'],dataset,identifier)
+        data['id'] = '{}/collections/{}/items/{}'\
+            .format(config['server']['url'], dataset, identifier)
     else:
-        data['id'] = '{}/collections/{}/items'.format(config['server']['url'],dataset)
+        data['id'] = '{}/collections/{}/items'\
+            .format(config['server']['url'], dataset)
 
     if data.get('timeStamp', False):
         data['https://schema.org/sdDatePublished'] = data.pop('timeStamp')
 
-    default_vocabulary = config['metadata']['default_vocabulary']
+    default_vocabulary = {
+         "geojson": "https://purl.org/geojson/vocab#",
+         "Feature": "geojson:Feature",
+         "FeatureCollection": "geojson:FeatureCollection",
+         "GeometryCollection": "geojson:GeometryCollection",
+         "LineString": "geojson:LineString",
+         "MultiLineString": "geojson:MultiLineString",
+         "MultiPoint": "geojson:MultiPoint",
+         "MultiPolygon": "geojson:MultiPolygon",
+         "Point": "geojson:Point",
+         "Polygon": "geojson:Polygon",
+         "bbox": {
+             "@container": "@list",
+             "@id": "geojson:bbox"
+         },
+         "coordinates": {
+             "@container": "@list",
+             "@id": "geojson:coordinates"
+         },
+         "features": {
+             "@container": "@set",
+             "@id": "geojson:features"
+         },
+         "geometry": "geojson:geometry",
+         "id": "@id",
+         "properties": "geojson:properties",
+         "type": "@type",
+         "description": "http://purl.org/dc/terms/description",
```

Comment:
What's the reason these are included?

Reply:
these = `description` and `title`

Reply:
Those two properties were a part of the vocabulary resolved by the original link.  We copied the whole vocab, should I omit these two properties?

Reply:
https://geojson.org/geojson-ld/geojson-context.jsonld

Reply:
https://geojson.org/geojson-ld/geojson-context.jsonld

Reply:
I don't see those properties there? That context ends at `type`, the one in your commit continues with `description` and `title`?

Reply:
Ah I see, at https://geojson.org/geojson-ld/ you can see a full exmaple, but the description and title parts are included since those are need for the example's `properties`. No, don't include those. Any additional context is expected to be specified in the configuration YAML.

Reply:
So locally this test fails in "test_api.py"  "test_get_collection_items_json_ld" without @type:
![image](https://user-images.githubusercontent.com/30809186/85804924-e6f2ee80-b718-11ea-907b-bc6839e6abe8.png)

Should I adjust this assertion?  The rest of the test works fine.

Reply:
Never mind my last comment, ha.